lib/cli/src/commands/app/logs.rs

```diff
@@ -131,9 +118,9 @@ impl crate::commands::AsyncCliCommand for CmdAppLogs {
 
         let logs_stream = wasmer_api::query::get_app_logs_paginated(
             &client,
-            name.clone(),
-            owner.to_string(),
-            version.clone(),
+            app.name.clone(),
+            app.owner.global_name.to_string(),
+            None, // keep version None since we want logs from all versions atm
```

Comment:
Can be done in a followup, but since yesterday there is a `AppIdent::AppVersion`, so we should respect that to  limit to a specific version in the future.

Reply:
I'll create a ticket for it, thanks!

Reply:
https://wasmerio.slack.com/archives/CT5U4HW2C/p1715856712738749

Reply:
https://github.com/wasmerio/wasmer/issues/4704